Sources/JavaScriptKit/JSObject.swift

```diff
@@ -57,4 +57,67 @@ public class JSObjectRef: Equatable {
     public func jsValue() -> JSValue {
         .object(self)
     }
+
+    public class func createTypedArray<Type>(_ array: [Type]) -> JSObjectRef where Type: TypedArrayElement {
```

Comment:
Why don't you provide new type interface like `class JSTypedArray<Element: JSTypedArrayElement>`  instead of publishing low level operation.